packages/react-scripts/scripts/start.js

```diff
@@ -152,6 +152,12 @@ checkBrowsers(paths.appPath, isInteractive)
         process.exit();
       });
     });
+
+    process.stdin.on('end', function() {
+      devServer.close();
+      process.exit();
+    });
+    process.stdin.resume();
```

Comment:
I suspect this line may be causing the CI failures

Reply:
At first pass, I'm unable to replicate the Travis CI environment locally. In manual testing, however, I can say that this line is necessary for the desired effect of exiting on stdin close. It may not be compatible with the `nohup` call in the testing script, however.

Reply:
Gotcha, thanks. I've confirmed this is the same logic `webpack-dev-server` uses when providing the `--stdin` flag: https://github.com/webpack/webpack-dev-server/blob/c9e9178a4882e414a6b9616baa35e8dbf7b2dd75/lib/utils/createConfig.js#L83-L90

Now we just need to figure out why CI is failing